libs/sdk-model/src/execution/executionDefinition/factory.ts

```diff
@@ -155,6 +156,20 @@ export function defWithSorting(definition: IExecutionDefinition, sorts: ISortIte
     return defSetSorts(definition, sorts);
 }
 
+/**
+ * Changes dateFormat in the definition.
+ *
+ * @param definition - execution definition to alter
```

Comment:
Please add a disclaimer simlar to the `defWithSorting` above that the function must be used in `withDateFormat`.